Examples/CapturingAndParsingPackets/Main.cs

```diff
@@ -1,25 +1,26 @@
 using System;
-using SharpPcap;
 using PacketDotNet;
+using SharpPcap;
```

Comment:
This change is effectively too large to review with almost 40k lines changed. For instance readonly modifiers sound great, using Int32 explicitly instead of int works etc, but its too much for me to review all at once.

Reply:
I am doing those changes with Resharper as an automated refactorization. The most I can do is to separate change of keyword type for CLR type to one commit, but it will be still in thousands of places in one commit. Is that acceptable, as long as it passes tests?